src/index.ts

```diff
@@ -10,26 +10,28 @@ import {
   PayInvoiceResponse,
   PaymentInfo,
   PaymentInfoOutgoing,
-  PhoenixConfig,
+  PhoenixConfig
```

Comment:
eslint rule comma dongle ( add comma to the end): 

```suggestion
  PhoenixConfig,
```